sir_trevor_rails.gemspec

```diff
@@ -36,6 +36,6 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "rails", ">= 3", "< 6"
   spec.add_dependency "redcarpet", ">= 2.0.1", "< 4"
-  spec.add_dependency "twitter-text", "~> 1.4"
+  spec.add_dependency "twitter-text", "~> 2.1.0"
```

Comment:
I guess if you do `~> 2.1` then any new minor versions would apply as well. If we only want patch versions then three digits is fine.